pleskdistup/common/tests/actionstests.py

```diff
@@ -44,6 +45,63 @@ def __init__(self, stages):
         super().__init__(stages, ".", flow_tracker=None)
 
 
+class SavedAction(action.ActiveAction):
+    def __init__(self):
+        self.name = "saved"
+        self.description = "Saved action description"
+
+    def _prepare_action(self) -> action.ActionResult:
+        return action.ActionResult()
+
+    def _post_action(self) -> action.ActionResult:
+        return action.ActionResult()
+
+    def _revert_action(self) -> action.ActionResult:
+        return action.ActionResult()
+
+
+def check_saved_state(phase, stage, saved_action_state, should_be_called):
+    simple_action = SimpleAction()
+    saved_action = SavedAction()
+
+    if phase == "prepare":
+        simple_action._prepare_action = mock.Mock(return_value=action.ActionResult())
```

Comment:
I suggest to use `setattr()` and `getattr()` to avoid duplicating these assignments.